test/contracts/EthToken.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity ^0.4.24;
 
 import "./TestToken.sol";
 
-contract WethToken is TestToken {
+contract EtherToken is TestToken {
```

Comment:
I think we don't need this anymore.
This is pooled already in `import "@gnosis.pm/dx-contracts/contracts/DxDevDependencies.sol";`

Reply:
We don't but I'd leave token contracts for now. Until wrong MGN migration is fixed.